### PR TITLE
Fix some issue in qa test

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve? 
fix some issue found in test.

### What is changed and how it works?
1. check backupTs > lastBackupTs 
2. set upper bound of backup concurrency otherwise it will cause tikv panic.
3. set `download file skip` log to warn.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

 - Need to cherry-pick to the release branch

### Release Note

 - No release note

### CODE REVIEW

File: pkg/task/backup.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ const (
 	flagGCTTL = "gcttl"
 
 	defaultBackupConcurrency = 4
+	defaultBackupMaxConcurrency = 128
```
Comment (kennytm) [inline pkg/task/backup.go:38]:
```suggestion
	maxBackupConcurrency = 128
```

128 sounds too small, as 128-core CPU is quite feasible.

File: pkg/task/backup.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ const (
 	flagGCTTL = "gcttl"
 
 	defaultBackupConcurrency = 4
+	maxBackupConcurrency     = 1024
```
Comment (overvenus) [inline pkg/task/backup.go:38]: This is too large, it will spawn 1024 thread, it could fail due to no resource (can not create threads). What about 128?

**Comment by 3pointer** [comment]
failed due to `dial tcp: lookup goproxy.onetool.net on 10.233.0.10:53: no such host`

**Pull request merged**